be/src/common/config.h

```diff
@@ -689,6 +689,8 @@ CONF_mInt64(experimental_s3_min_upload_part_size, "16777216");
 
 CONF_Int64(max_load_dop, "16");
 
+CONF_Int64(rocksdb_manual_compact_live_sst_files_size, "10737418240"); // 10G
```

Comment:
meta_threshold_to_manual_compact is a better name. Prefix rocksdb may be a misunderstanding.

Reply:
changed.